opencog/guile/SchemePrimitive.h

```diff
@@ -139,7 +139,7 @@ class SchemePrimitive : public PrimitiveEnviron
 			Q_H,   // return HandleSeq, take handle
 			Q_HTI, // return HandleSeq, take handle, type, and int
 			Q_HTIB,// return HandleSeq, take handle, type, and bool
-			K_H,   // return HandleSeqSeq, take Handle
+			K_HI,  // return HandleSeqSeq, take Handle, int
```

Comment:
Because sureal is the only one using K_H, so I believe this should not affect the others